engine.py

```diff
@@ -313,6 +313,15 @@ def make_move(self, move):
         self.move_log.append(move) # log move
         self.light_to_move = not self.light_to_move # next player to move
 
+        # Pawn Promotion
+        if move.isPawnPromotion:
+            promotedPiece = input("Promote to q, r, b, or n ") #we can add this to the ui later
```

Comment:
Make the input prompt more descriptive, something like;
input("Promote pawn to: Queen (q), rook (r), bishop (b), knight (n)")